packages/plugins/rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec

```diff
@@ -113,6 +113,15 @@ mv %{buildroot}%{gem_instdir}/settings.d/remote_execution_ssh.yml.example \
 %doc %{gem_instdir}/README.md
 
 %changelog
+* Tue Nov 14 2023 Adam Ruzicka <aruzicka@redhat.com> 0.10.3-1
+- Update to 0.10.3
+
+* Fri Oct 27 2023 Adam Ruzicka <aruzicka@redhat.com> 0.10.2-2
```

Comment:
you didn't pick all the history, and now the 0.10.1 gem is still in git, confusing git annex.

either cherry pick all develop commits, or only update to 0.10.3 :)